### DISCUSSION TITLE
Updated discord link?

### DISCUSSION THREAD

**Discussion by bruor** [seerr-team/seerr#518] [General]
The link in the readme is expired, can someone throw an invite link in here please?

**Comment by fallenbagel** [comment]
I just checked and it doesn't seem to be expired. 

But still heres a new one: https://discord.com/invite/BEstYE6eNS

**Reply by bruor** [reply]
Oh, the links don't work in the browser if I don't have the desktop app for discord open...

**Reply by fallenbagel** [reply]
I tried on browser and it redirected to discord.com/[link]

**Reply by bruor** [reply]
If I don't have discord open and I try the link in the readme I see this
![image](https://github.com/Fallenbagel/jellyseerr/assets/454516/e649c409-47be-4eb1-8d0b-12057a85a303)

But if I have the app open and I hit the link I get this: 
![image](https://github.com/Fallenbagel/jellyseerr/assets/454516/63b4d784-5e71-4a50-a510-a21264938142)

Apparently I'm just a bit of a newb with discord :D

**Comment by apennismightier** [comment]
So is there a new link for it that actually works or not? I can't get in on my browser or through the app on desktop or mobile.

**Reply by fallenbagel** [reply]
I just saw a bunch of people jump in using the same link in the readme. Are you trying the link here? You should be using the link in readme.

**Reply by apennismightier** [reply]
Just got your email with the invite. It only worked on my phone but I'm in. Ty!
